### PR TITLE
LRAC-13837 Render <a> tag for external links

### PR DESCRIPTION
**Pull request by interaminense**
JIRA Issue: https://issues.liferay.com/browse/LRAC-13837

### CODE REVIEW

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/user-dropdown/Item.tsx [hunk 1]
```diff
@@ -34,14 +43,20 @@ const UserMenuDropdownItem: React.FC<
...
-				<ClayLink
-					block
-					className='button-root'
-					displayType='unstyled'
-					href={url}
-				>
-					<Content />
-				</ClayLink>
+				externalLink ? (
+					<a href={url}>
```
Comment (ilzamcmed) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/user-dropdown/Item.tsx:47]: Hey @interaminense 
In this case, should we add a `target="_blank"` attribute?

**Review by ilzamcmed — APPROVED** [review]
LGMT!

**Comment by interaminense** [comment]
ci:test:relevant

**Review by JLucasM — APPROVED** [review]
LGTM!

**Comment by interaminense** [comment]
ci:forward:force

**Pull request closed without merging**